### PR TITLE
upstream sync

### PR DESCRIPTION
**Pull request by baywet**
- microsoft graph update: dotnet core migration (#65)
- Clean-Up (#66)
- Properly handle enumerations headers that contain part of reserved title word  (#64)
- Support recursive models & Clean up (#69)
- sync to upstream (#70)
- Look up subNamespaces in Metadata route (#73)
- Handle missing OData type in response metadata (#76)
- Add validation error for empty baseType value in resource (#74)
- Convert error to warning for complex types with navigation properties (#75)
- Allow complex types to have navigation properties (#78)
- Add error for table that we think holds enum values but has wrong column name (#77)
- Add more information in error message when function call does not have key-value params (#85)
- Ensure JSON request or response body is not whitespace (#87)
- Flag resource types specified in response metadata but response has no body (#86)
- add missing project URL (#93)
- Handle user error when namespace is not specified in resource metadata @data.type (#95)

### CODE REVIEW

**Comment by BarryShehadeh** [comment]
Looks good to me.

File: ApiDoctor.Console/Program.cs [hunk 1]
```diff
@@ -1854,7 +1854,7 @@ private static async Task<bool> CheckServiceMetadataAsync(CheckMetadataOptions o
                         {
                             RelaxedStringValidation = true,
                             IgnorablePropertyTypes = metadataValidationConfigs?.IgnorableModels,
-                            AllowTruncatedResponses = modelConfigs?.SkipProprtiesValidation ?? false
+                            AllowTruncatedResponses = modelConfigs?.TruncatedProprtiesValidation ?? false
```
Comment (millicentachieng) [inline ApiDoctor.Console/Program.cs:1857]: NIT: Missing letter `TruncatedPropertiesValidation`

File: ApiDoctor.Validation/Config/MetadataValidationConfigFile.cs [hunk 2]
```diff
@@ -59,8 +59,8 @@ public class ModelConfigs
         [JsonProperty("aliasNamespace")]
         public string AliasNamespace { get; set; }
 
-        [JsonProperty("skipProprtiesValidation")]
-        public bool SkipProprtiesValidation { get; set; }
+        [JsonProperty("truncatedProprtiesValidation")]
+        public bool TruncatedProprtiesValidation { get; set; }
```
Comment (millicentachieng) [inline ApiDoctor.Validation/Config/MetadataValidationConfigFile.cs:63]: NIT: Missing letter `TruncatedPropertiesValidation`

File: ApiDoctor.Console/Program.cs (hunk 1 above)
Comment (baywet) [reply ApiDoctor.Console/Program.cs:1857]: Thanks for noticing this. As this is a sync pr, I'm not going to make any changes.
We could address that in a separate pr, or pr the upstream. Do you want to create an issue here so we tackle that before merging back to upstream?

Comment (baywet) [reply ApiDoctor.Console/Program.cs:1857]: https://github.com/OneDrive/apidoctor/issues/101

File: ApiDoctor.Validation/Config/MetadataValidationConfigFile.cs (hunk 2 above)
Comment (baywet) [reply ApiDoctor.Validation/Config/MetadataValidationConfigFile.cs:63]: https://github.com/OneDrive/apidoctor/issues/101

**Pull request merged**